compiler/rustc_codegen_ssa/src/lib.rs

```diff
@@ -96,6 +109,10 @@ pub struct CompiledModule {
     pub object: Option<PathBuf>,
     pub dwarf_object: Option<PathBuf>,
     pub bytecode: Option<PathBuf>,
+
+    /// items created to satisfy --emit
```

Comment:
This doc comment only documents the first field, not both. Either turn this into a normal comment or drop it entirely (it's not *super* useful imho).